test/integration/spec/localparticipant.js

```diff
@@ -29,7 +30,12 @@ const defaults = require('../../lib/defaults');
 const { isChrome, isFirefox, isSafari } = require('../../lib/guessbrowser');
 const { createRoom, completeRoom } = require('../../lib/rest');
 const getToken = require('../../lib/token');
-const { isRTCRtpSenderParamsSupported, smallVideoConstraints } = require('../../lib/util');
+
+const {
+  createSyntheticAudioStreamTrack,
```

Comment:
@manjeshbhargav - I suggest we keep these bug fix PRs very specific to the bug. Are the other changes in tests necessary for this change?

Reply:
@makarandp0 Not directly, but since our existing tests did not catch this issue, I wanted to make sure that tests which are related to calling publishTrack multiple times should cover LocalTracks of all kinds.